cmd/core-command/res/configuration.yaml

```diff
@@ -18,7 +18,6 @@ Clients:
     Protocol: http
     Host: localhost
     Port: 59881
```

Comment:
This needs the SecurityOptions since this client is not in common config for Core/Support services.

Reply:
added